pkgs/flox-bash/default.nix

```diff
@@ -15,18 +15,7 @@
   glibcLocalesUtf8,
   darwin,
 }: let
-  nixPatched = nixVersions.nix_2_15.overrideAttrs (oldAttrs: {
-    patches =
-      (oldAttrs.patches or [])
-      ++ [
-        ./nix-patches/CmdProfileBuild.patch
-        ./nix-patches/CmdSearchAttributes.patch
-        ./nix-patches/update-profile-list-warning.patch
-        ./nix-patches/multiple-github-tokens.2.13.2.patch
-        ./nix-patches/curl_flox_version.patch
-        ./nix-patches/no-default-prefixes-hash.2.15.1.patch
-      ];
-  });
+  nixPatched = nixVersions.nix_2_15;
```

Comment:
nit:
```suggestion
  nix = nixVersions.nix_2_15;
```

Reply:
Thanks. Note that we'll change this to use the nix provided by the overlay as soon as https://github.com/flox/pkgdb/pull/113 is merged and changes are bumped through.